index.js

```diff
@@ -1,14 +1,15 @@
 const ethUtil = require('ethereumjs-util')
 const rlp = require('rlp')
+const Buffer = require('safe-buffer').Buffer
 
 var Account = module.exports = function (data) {
   // Define Properties
   var fields = [{
     name: 'nonce',
-    default: new Buffer([])
+    default: Buffer.from([])
```

Comment:
Buffer.allocUnsafe(0) ?

Reply:
Thanks.

Sorry, I'm not really aware of the implications, can someone explain? Is this only for ``nonce``?

Reply:
`Buffer.from([])` returns an empty buffer. `Buffer.allocUnsafe(0)` does the same but is faster.

Reply:
My intentions mostly was that `alloc*(0)` is more explicit than `.from([])`

Reply:
So maybe we should avoid using the ``unsafe`` version all together in ``ethereumjs`` context, for not risking to leak a private key or stuff like this?@axic ?

Just tested both ``alloc`` versions on the console, there was not some noticeable lag on the output, feels like it doesn't really make a difference - at least for one-time operations.

Reply:
Replaced this with ``alloc``, actually feel better with this.

Reply:
You did not see the difference, because `alloc(0)` and `allocUnsafe(0)` return same.
alloc: https://github.com/nodejs/node/blob/28a0af358ac998f94052199dc62a8140e8a11ac5/lib/buffer.js#L269
allocUnsafe: https://github.com/nodejs/node/blob/28a0af358ac998f94052199dc62a8140e8a11ac5/lib/buffer.js#L278 & https://github.com/nodejs/node/blob/28a0af358ac998f94052199dc62a8140e8a11ac5/lib/buffer.js#L307

Reply:
`alloc(0)` will be the same since there is 0 bytes to clean after allocation.

Reply:
@fanatid @axic Ok, to sum this up: I've learned a lot and we can leave ``alloc(0)`` since it doesn't make a difference both on the security and the speed side? :-)

Reply:
I don't mind which is used, just make sure to update -util and -vm accordingly. -vm only uses `alloc`, -util uses `allocUnsafe` also in a combination of `allocUnsafe`+`fill` (which should really be just `alloc`).